tools/report-converter/codechecker_report_converter/twodim.py

```diff
@@ -37,17 +37,17 @@ def to_str(
 
     if format_name == 'rows':
         return to_rows(rows)
-    elif format_name == 'table' or format_name == 'plaintext':
+    if format_name in ('table', 'plaintext'):
         # TODO: 'plaintext' for now to support the 'CodeChecker cmd' interface.
         return to_table(all_rows, True, separate_footer)
-    elif format_name == 'csv':
+    if format_name == 'csv':
         return to_csv(all_rows)
-    elif format_name == 'dictlist':
+    if format_name == 'dictlist':
         return to_dictlist(keys, rows)
-    elif format_name == 'json':
+    if format_name == 'json':
         return json.dumps(to_dictlist(keys, rows))
```

Comment:
I think elif statement would be prettier.

Reply:
Same answer.